Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1848,17 +1850,21 @@ namespace OpenInfraPlatform {
 				) const throw(...)
 				{
 					carve::geom::vector<3> centerToTrimPoint = trimPoint - circleCenter;
-					if (abs(centerToTrimPoint.length() - circleRadius) < 0.0001) {
+
+					if (GeomSettings()->areEqual(abs(centerToTrimPoint.length()), circleRadius)){
 						centerToTrimPoint.normalize();
 						double cosAngle = carve::geom::dot(centerToTrimPoint, carve::geom::vector<3>(carve::geom::VECTOR(1., 0., 0.)));
 
-						if (abs(cosAngle) < 0.0001) {
+						if (GeomSettings()->areEqual(abs(cosAngle), 0.)) {

```

Comment:
Always call `this->GeomSetting()...` - it is an error with vs2019, which we will be switching to shortly.

Reply:
Resolved: 6e54eae